src/sql/parts/profiler/contrib/profilerActions.ts

```diff
@@ -98,12 +100,22 @@ export class ProfilerPause extends Action {
 	}
 
 	public run(input: ProfilerInput): TPromise<boolean> {
-		this.enabled = false;
 		return TPromise.wrap(this._profilerService.pauseSession(input.id).then(() => {
```

Comment:
@kburtram I agree with your earlier comment that pauseSession should be renamed to togglePauseSession, but for the sake of this PR it broke a lot of communication between tools service and sql ops. I'll be including that change in a future PR, along with some changes to startSession. If you think that's important to have in this commit I can put that change in.